src/components/editor/property/PropertyPanel.jsx

```diff
@@ -23,7 +23,7 @@ export default class PropertyPanel extends Component {
 
   render() {
     return (
-      <div className={this.getCssClasses()}>
+      <div className={this.getCssClasses()} propLabel={this.props.pt.propertyLabel}>
```

Comment:
adding this propLabel attribute makes CSS selectors for integration tests a lot more clear ...